packages/tools/locales/es_ES.po

```diff
@@ -43,10 +45,10 @@ msgstr "OK"
 #: packages/app-desktop/bridge.js:110 packages/app-desktop/bridge.js:126
 #: packages/app-desktop/checkForUpdates.js:154
 #: packages/app-desktop/gui/DialogButtonRow.min.js:38
-#: packages/app-desktop/gui/SideBar/SideBar.js:201
-#: packages/app-desktop/gui/SideBar/SideBar.js:430
+#: packages/app-desktop/gui/SideBar/SideBar.js:191
+#: packages/app-desktop/gui/SideBar/SideBar.js:412
 #: packages/app-desktop/gui/KeymapConfig/ShortcutRecorder.js:52
-#: packages/app-desktop/gui/utils/NoteListUtils.js:165
+#: packages/app-desktop/gui/utils/NoteListUtils.js:164
 #: packages/app-desktop/gui/ConfigScreen/ConfigScreen.js:528
```

Comment:
I have to fix these and others like them too, right?

Reply:
Yes, but maybe I can fix it quickly. I'll have a look later this evening.

The fact that you didn't use a branch makes this a bit complicated. What you usually would do is rebase to dev HEAD.